vega-lite-schema.json

```diff
@@ -1025,6 +1025,12 @@
             },
             "type": "object"
         },
+        "Dict<SelectionSpec>": {
```

Comment:
Looks like we need other ways to declare this or need @domoritz to modify the schema generator?